### DISCUSSION TITLE
Copilot causing Jetbrains IDEs to be uninteractlve

### DISCUSSION THREAD

**Discussion by BeauTheBeau** [community/community#75611] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot 1.4.0.3748 causes JetBrains WebStorm 2023.2.5 + 2023.2.4 and PyCharm Professional 2023.2.5 to become unresponsive to user input, with the loading spinner in the center. I assume this applies to all other versions, but I'm unable to check.

![image](https://github.com/community/community/assets/90942493/9c42379f-9e69-47a4-acb7-c0a19653fa8c)

Here is the log file from Pycharm: [idea.log](https://github.com/community/community/files/13365746/idea.log)

**Comment by Akash1134** [comment]
Hi @BeauTheBeau ,

Thanks for getting in touch.

Can you tell us whether or not you're using a virtual machine, or are you using NAT or Bridge to connect your network?

Please also check in the IDE Console tab if there any output for GitHub Copilot in the output panel.

**Reply by BeauTheBeau** [reply]
Hey! 

I'm not sure exactly what happened, but this issue has magically resolved itself, will answer your questions regardless :)


> whether or not you're using a virtual machine

I'm not running in a virtual machine, using bare metal. 


> are you using NAT or Bridge to connect your network

My PC is connected to my phone's mobile hotspot, I don't currently have a working router and am therefore using my mobile data haha. I believe that'd be over NAT.


> check in the IDE Console tab

Other than the idea.log I uploaded, I have nothing else to provide, unfortunately
